Makefile

```diff
@@ -205,6 +207,12 @@ install: all
 	$(Q)$(INSTALL) -d -m 755 $(DESTDIR)$(bindir)
 	$(Q)$(INSTALL) -d -m 755 $(DESTDIR)$(libdir)
 	$(Q)$(INSTALL) -d -m 755 $(DESTDIR)$(etcdir)/bash_completion.d
+ifneq ($(wildcard $(elfdir)/lib/libelf.so),)
```

Comment:
please add a comment that it only installs when it's built but not installed yet.

Reply:
Done. Thanks!